src/mycollections/DynamicArray.java

```diff
@@ -1,59 +1,143 @@
 package mycollections;
 
-public class DynamicArray<E>  implements MyList<E> {
-
-	@Override
-	public int size() {
-		return 0;
-	}
-
-	@Override
-	public boolean isEmpty() {
-		return false;
-	}
-
-	@Override
-	public boolean contains(E element) {
-		return false;
-	}
-
-	@Override
-	public int indexOf(E element) {
-		return 0;
-	}
-
-	@Override
-	public int lastIndexOf(E element) {
-		return 0;
-	}
-
-	@Override
-	public E get(int index) {
-		return null;
-	}
-
-	@Override
-	public E set(int index, E element) {
-		return null;
-	}
-
-	@Override
-	public boolean add(E e) {
-		return false;
-	}
-
-	@Override
-	public void add(int index, E element) {
-
-	}
-
-	@Override
-	public E remove(int index) {
-		return null;
-	}
-
-	@Override
-	public boolean remove(E element) {
-		return false;
-	}
+public class DynamicArray<E> implements MyList<E> {
+
+    int size = 0;
+    int cursor = -1;
+    private static final int DEFAULT_CAPACITY = 10;
+    E[] arrayList;
+
+    public DynamicArray(int capacity) {
+        this.size = size;
```

Comment:
There is no reason to re-assign a variable to itself. Either this statement is redundant and should be removed, or the re-assignment is a mistake and some other value or variable was intended for the assignment instead.